doc/1/controllers/realtime/count/snippets/count.cs

```diff
@@ -0,0 +1,15 @@
+NotificationHandler listener = (response) => {};
+
+try {
+  string room_id = await kuzzle.Realtime.SubscribeAsync(
+    "nyc-open-data",
+    "yellow-taxi",
+    JObject.Parse("{}"),
+    listener);
+
+  int count = await kuzzle.Realtime.CountAsync(room_id);
+
+  Console.WriteLine("Currently " + count + " active subscription");
```

Comment:
Use string litterals:

```suggestion
  Console.WriteLine($"Currently {count} active subscription");
```

Reply:
This change should be applied to all code snippets.